checkov/cloudformation/runner.py

```diff
@@ -35,8 +35,9 @@ def __init__(
         source: str = "CloudFormation",
         graph_class: Type[LocalGraph] = CloudformationLocalGraph,
         graph_manager: Optional[GraphManager] = None,
-        external_registries: Optional[List[BaseRegistry]] = None,
+        external_registries: Optional[List[BaseRegistry]] = None
     ) -> None:
+        super().__init__(file_extensions=['.json', '.yml', '.yaml'])
```

Comment:
@mikeurbanski1 Should add `.template` at least, I believe cfn doesn't require a specific extension...